components/UI/navbar/navbar.module.scss

```diff
@@ -0,0 +1,55 @@
+.infoNavbar nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #1d1283;
+}
+.infoNavbar nav li {
+  float: left;
+}
+.infoNavbar nav li a {
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  font-weight: bolder;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+.infoNavbar nav li a:hover {
+  background-color: rgba(255, 255, 255, 0.2);
+}
+.infoNavbar nav{
+  background: #1d1283;
+}
+.DrawerToggle {
+  position: relative;
+  width: 35px;
+  height: 70%;
+  left: 10px;
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  align-items: center;
+  padding: 10px 0;
+  box-sizing: border-box;
+  cursor: pointer;
+  display: flex;
+  display: none;
```

Comment:
Could you please re-check the css here, I can see `display` property 3 times.

Reply:
@sumitd94 repeating CSS property has been removed. Please check.

Reply:
I can still see a `display` property at line no 31

Reply:
@sumitd94 display property at line no 31 has been removed. Please check.